code/controllers/subsystem/ticker.dm

```diff
@@ -500,6 +500,8 @@ SUBSYSTEM_DEF(ticker)
 		else
 			stack_trace("[S] [S.type] found in start landmarks list, which isn't a start landmark!")
 
+		if(!rulermob)
+			lord_color_default()
```

Comment:
Is this really only ran 10m into the game? I don#t really mind if it's not, but I'd like to get this clarified

Reply:
*Shrugs*, I kinda disabled that function because it was making conflicts, that thing was probably meant to disallow changing of default realm colors if you were 10 minutes late into the game, it most likely serves no actual purpose now, but it doesn't cause conflict, so, I ain't touching that.

Reply:
Disabled what in particular?